browse/static/js/catalyzex.js

```diff
@@ -59,4 +82,20 @@
     $output.append(`<p>No code found for this paper just yet.</p>`)
   }
   $output.append(`<p>If you have code to share with the arXiv community, please ${submitItHereLink} to benefit all researchers & engineers.</p>`)
+  
+  if(isAlertActive) {
+    $output.append(`<p>You have created an alert for this paper and will be notified when new code is available 🔔</p>`)
```

Comment:
@riltonfranzonee do we want to add a link for seeing all user added alerts here, like we were doing in the extension as well ?

Reply:
done (added a screenshot below) ✅